frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1281,6 +1281,15 @@ export const surveyLogic = kea<surveyLogicType>([
                             question: !question.question && 'Please enter a question label.',
                         }
 
+                        if (question.type === SurveyQuestionType.Link) {
+                            if (!question.link?.startsWith('https://') && !question.link?.startsWith('mailto:')) {
```

Comment:
should we rather try to create an instance of URL instead? a non https is still a valid URL.

Reply:
it's a [BE rule to only allow https:// or mailto](https://github.com/PostHog/posthog/blob/master/posthog/api/survey.py#L52)

Reply:
Mmm ok, i still think we should create a url which validates if its a valid url, otherwise https://$&12/.89 is a "valid" but its not really

Reply:
makes sense, added better validation here: https://github.com/PostHog/posthog/pull/28762/commits/97f1d56c8d7287d60ddfa1cca286b5eaf7297c9b